nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/canvas/flow-analysis-drawer.jsp

```diff
@@ -0,0 +1,85 @@
+<%--
+ Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+--%>
+<%@ page contentType="text/html" pageEncoding="UTF-8" session="false" %>
+<section id="flow-analysis-drawer">
+    <div class="flow-analysis-header">
+        <div class="flow-analysis-refresh-container">
+            <button id="flow-analysis-check-now-btn" class="flow-analysis-check-now-btn">Start a new analysis</button>
```

Comment:
When the user clicks this button there is no feedback. With no feedback, I was questioning whether it worked or not. As I repeatedly click on the link, new flow analysis reports seem to be generated. At a minimum, we should probably limit a client from having more than one outstanding request.

Alternatively, should be feature be simplified and the UI only show whatever the current results are? If so, we could just remove this link.

Reply:
I updated my work to disable this button and show a spinner while a report is being run.